### ISSUE TITLE
CLI issue with python <= 3.6

### ISSUE THREAD

**Issue by jprevc** [netioproducts/PyNetio#5]
Using python <= 3.6, calling 

`$ Netio -h `

from command line produces the following output:
```
Internal error:  __init__() got an unexpected keyword argument 'required'
Traceback (most recent call last):
  File "D:\PREVC\GIT\PyNetio\Netio\cli.py", line 227, in main
    args = parse_args()
  File "D:\PREVC\GIT\PyNetio\Netio\cli.py", line 186, in parse_args
    command_parser = parser.add_subparsers(metavar="COMMAND", help="device command", required=True)
  File "C:\iSYSTEM\winIDEA9\Python\lib\argparse.py", line 1703, in add_subparsers
    action = parsers_class(option_strings=[], **kwargs)
TypeError: __init__() got an unexpected keyword argument 'required'
```

Problem occurs due to incompatibility of argparse's ArgumentParser.add_subparsers function, which did not have 'required' argument in python 3.6:
https://docs.python.org/3.6/library/argparse.html#argparse.ArgumentParser.add_subparsers

but added it in python 3.7:
https://docs.python.org/3.7/library/argparse.html#argparse.ArgumentParser.add_subparsers

The solution would be to change line:
`command_parser = parser.add_subparsers(metavar="COMMAND", help="device command", required=True)`

to 
`command_parser = parser.add_subparsers(metavar="COMMAND", help="device command")`

Or alternatively, use workaround:
https://stackoverflow.com/a/22994500

**Comment by AdamVerner** [comment]
Thanks for the report and explanation.
Python3.6 is getting a little bit old now and won't receive security updates in 2022, how the time flies!

Anyway the fix is simple so i see no problem in adding it.

**Comment by AdamVerner** [comment]
Fixed in version 1.0.6 which should now be live on PyPI, don't forget to `pip install --upgrade Netio`
